src/cholesky.jl

```diff
@@ -0,0 +1,65 @@
+import Base: getproperty
+import LinearAlgebra: cholesky, Cholesky, char_uplo, UpperTriangular, LowerTriangular, \,
+    istril, istriu
+
+const KroneckerCholesky{T} = Cholesky{T, <:AbstractKroneckerProduct{T}} where {T}
+
+function cholesky(A::AbstractKroneckerProduct; check=true)
+    P, Q = getmatrices(A)
+    chol_P, chol_Q = cholesky(P; check=true), cholesky(Q; check=true)
```

Comment:
should here not be `cholesky(P; check=check)` or did you intend to always perform the check?

Reply:
Totally right, should have been propagating `check`. I'll let you resolve on your cholesky branch.